### PR TITLE
Add company icon on contact tabs for direct link

### PR DESCRIPTION
**Pull request by defrance**
very usefull for creating new lefted contact

### CODE REVIEW

**Comment by eldy** [comment]
Can you solve conflit ?

**Comment by defrance** [comment]
conflict solved

File: htdocs/core/tpl/contacts.tpl.php [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?php
-/* Copyright (C) 2012      Regis Houssin       <regis.houssin@capnetworks.com>
- * Copyright (C) 2013-2014 Laurent Destailleur <eldy@users.sourceforge.net>
+/* Copyright (C) 2012      Regis Houssin        <regis.houssin@capnetworks.com>
+ * Copyright (C) 2013-2014 Laurent Destailleur  <eldy@users.sourceforge.net>
+ * Copyright (C) 201	   Charlie BENKE 	<charlie@patas-monkey.com>
```
Comment (rdoursenaud) [inline htdocs/core/tpl/contacts.tpl.php]: I think you miss a digit here ;)

**Comment by defrance** [comment]
change done for the lefted digit

**Pull request merged**